### PR TITLE
Add jaeger support for ctxtags extraction

### PR DESCRIPTION
**Pull request by vporoshok**

### CODE REVIEW

**Comment by vporoshok** [comment]
I signed it!

**Comment by vporoshok** [comment]
Yeah, sure! Give few minutes!

**Comment by domgreen** [comment]
Thank you 😀

File: tracing/opentracing/id_extract.go [hunk 1]
```diff
@@ -38,5 +40,14 @@ func (t *hackyTagsCarrier) Set(key, val string) {
 		t.Tags.Set(TagTraceId, val) // this will most likely be base-16 (hex) encoded
 	} else if (strings.Contains(key, "spanid") && !strings.Contains(key, "parent")) || (strings.Contains(strings.ToLower(key), "spanid") && !strings.Contains(strings.ToLower(key), "parent")) {
 		t.Tags.Set(TagSpanId, val) // this will most likely be base-16 (hex) encoded
+	} else if key == "uber-trace-id" {
+		parts := strings.Split(val, ":")
+		if len(parts) == 1 {
+			// for extraordinary cases
```
Comment (jutkko) [inline tracing/opentracing/id_extract.go]: Extraordinary cases? Maybe special cases?

Comment (domgreen) [reply tracing/opentracing/id_extract.go]: Under what circumstance would this happen? (I'm not very familiar with Jaeger)

Comment (vporoshok) [reply tracing/opentracing/id_extract.go]: I have not encountered such cases, so I left this comment

Comment (vporoshok) [reply tracing/opentracing/id_extract.go]: Is it ok if only normal case would be handled?

**Comment by vporoshok** [comment]
thank you for your patience, I hope that I managed to find optimal solution for testing this functionality

**Comment by vporoshok** [comment]
Thank you!

**Pull request merged**